README.md

```diff
@@ -32,6 +32,7 @@ idl::Context context = idl::parse(my_idl);
 const StructType& inner = context.module().structure("Inner");
 const StructType& outer = context.module().structure("Outer");
 ```
+More details about the parser usage can be found at [the parser section](#parser).
```

Comment:
I think that at this moment this line can be too many. I mean, Maybe only we can catch the user for only ten seconds. We need only the essential info at this moment. I would put the link over _parsing the IDL_ of the previous line `29`.